projects/configs/tag-operations/base.yaml

```diff
@@ -0,0 +1,660 @@
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180094
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/mr-spectroscopy-multi-frame-functional-groups/52009229/00189103/00180094
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009229/00189125/00180094
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189125/00180094
+- name: "Software Version"
+  group: 0x0018
+  element: 0x1020
+  op: "keep"
+#https://dicom.innolitics.com/ciods/ct-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/mr-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/ultrasound-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/rt-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/digital-x-ray-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/digital-intra-oral-x-ray-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/digital-mammography-x-ray-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/x-ray-3d-angiographic-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/x-ray-3d-craniofacial-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/breast-projection-x-ray-image/general-equipment/00181020
+#https://dicom.innolitics.com/ciods/enhanced-continuous-rt-image/general-equipment/00181020
+#NOTE The search of "00181020" found 579 results
+- name: "Field Of View Dimension"
+  group: 0x0018
+  element: 0x1149
+  op: "keep"
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/dx-detector/00181149
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/dx-detector/00181149
+#https://dicom.innolitics.com/ciods/x-ray-radiofluoroscopic-image/x-ray-acquisition/00181149
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/x-ray-acquisition/00181149
+#https://dicom.innolitics.com/ciods/digital-x-ray-image/dx-detector/00181149
+#https://dicom.innolitics.com/ciods/digital-mammography-x-ray-image/dx-detector/00181149
+#https://dicom.innolitics.com/ciods/digital-intra-oral-x-ray-image/dx-detector/00181149
+- name: "Imager Pixel Spacing"
+  group: 0x0018
+  element: 0x1164
+  op: "keep"
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/dx-detector/00181164
+#https://dicom.innolitics.com/ciods/x-ray-radiofluoroscopic-image/x-ray-acquisition/00181164
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/x-ray-acquisition/00181164
+#https://dicom.innolitics.com/ciods/digital-x-ray-image/dx-detector/00181164
+#https://dicom.innolitics.com/ciods/digital-mammography-x-ray-image/dx-detector/00181164
+#https://dicom.innolitics.com/ciods/digital-intra-oral-x-ray-image/dx-detector/00181164
+- name: "Grid"
+  group: 0x0018
+  element: 0x1166
+  op: "keep"
```

Comment:
is this only x-ray? Should consider if we want to have an x-ray specific file if so

Reply:
[7b66165](https://github.com/SAFEHR-data/PIXL/pull/498/commits/7b661657a75b7753d41a3bc4dfd7edead4b547bc)